_im-plugin/ism/policies.md

```diff
@@ -438,6 +439,133 @@ Parameter | Description | Type | Required
 ]
 ```
 
+### rollup
+
+[Index rollup]({{site.url}}{{site.baseurl}}/im-plugin/index-rollups/index/) lets you periodically reduce data granularity by rolling up old data into summarized indices.
+
+Only a single rollup job can be created with this method. The source index for the rollup job is inferred from the index the policy it is attached to.
```

Comment:
Rollup job can be continuous or non-continuous/one time, the rollup job created using ISM can only be non-continuous

Reply:
Thanks, I'll get that updated